app/assets/stylesheets/modules/_timeline.styl

```diff
@@ -131,11 +131,12 @@
 
 .block__block-type
   color $text_dk
-  font-size 14px
+  font-size 14  px
```

Comment:
Is that a typo?

Reply:
Yeah, sorry I didn't notice it. 
I have fixed it.